corpora/proverbs.dep

```diff
@@ -27,3 +28,979 @@
 "<.>"
 	"." sent @punct #12->11
 
+# sent_id = proverbs:1
+# author:id = Сыргабековна&Сабыровна:329
+# text = Миң сөз айтканча, бир ишти бүтүр.
+# text[eng] = [Instead of saying] [a thousand] [words], finish one task.
+# label_cg = JNW_checked
+# label_dep = JNW_initial_checked
```

Comment:
What's the difference between `label_text`, `label_cg`, `label_dep`, and `labels`?